packages/slice-machine/src/utils/tracker.ts

```diff
@@ -0,0 +1,57 @@
+import { createContext } from "react";
```

Comment:
I wondering shouldn't we have a common Tracker file in the Lib folder ?
The difference I see is that front-end event are not tracked by user as we don't seem to know who this is.
And back-end events can be tracked by users.

I thought the point of the anonymousId (which I don't see here, so i'm guessing it's sent by segment when we track and event) was to be anonymous until we actually login once.

But here it seems that the events are tracked anonymously even if we are logged in already.

Reply:
They are because segment send a cookie with the Id along with the request on client side.
Look for 'resolveUser' that will attach a userId to the anonymousID considered as the current sessionId basically